samples/shopping-cart-with-view/src/test/java/shoppingcart/ShoppingCartIntegrationTest.java

```diff
@@ -0,0 +1,62 @@
+package shoppingcart;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import akka.javasdk.testkit.TestKitSupport;
+import shoppingcart.application.ShoppingCartEntity;
+import shoppingcart.domain.ShoppingCartState;
+
+public class ShoppingCartIntegrationTest extends TestKitSupport {
```

Comment:
Heh, still not formatted as other samples:
<img width="301" alt="image" src="https://github.com/user-attachments/assets/ba05baab-e386-4b7c-8ad8-9f0e056afd87" />

It's useful to have a small indentation for docs

Reply:
I don't know what's wrong with this. I'm not using vscode, and the java language server is just losing its mind trying to format this one file.